specification/migrate/resource-manager/Microsoft.Migrate/stable/2020-05-01/hubmigrate.json

```diff
@@ -0,0 +1,1101 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-05-01",
+    "title": "Microsoft.Azure.AMH.WebRole.V2020_05_01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "parameters": {
+    "subscriptionId": {
+      "name": "subscriptionId",
+      "description": "Azure Subscription Id in which project was created.",
+      "type": "string",
+      "in": "path",
+      "required": true
+    },
+    "resourceGroupName": {
+      "name": "resourceGroupName",
+      "description": "Name of the Azure Resource Group that project is part of.",
+      "type": "string",
+      "in": "path",
+      "x-ms-parameter-location": "method",
+      "required": true
+    },
+    "migrateProjectName": {
+      "name": "migrateProjectName",
+      "description": "Migrate project name.",
+      "in": "path",
+      "x-ms-parameter-location": "method",
+      "required": true,
+      "type": "string"
+    }
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Migrate/migrateProjects/{migrateProjectName}": {
```

Comment:
Please check Swagger Lint Diff and fix below errors.

R4015 - NestedResourcesMustHaveListOperation
R4016 - TopLevelResourcesListByResourceGroup
R4017 - TopLevelResourcesListBySubscription
R4036 - ImplementPrivateEndpointAPIs